lib/components/popover/tooltip-controller.ts

```diff
@@ -1,5 +1,5 @@
-import * as Stacks from "../stacks";
-import { BasePopoverController, PopoverController } from "./s-popover";
+import * as Stacks from "../../stacks";
+import { BasePopoverController, PopoverController } from "./controller";
```

Comment:
Tooltip is tricky since, from a styling viewpoint, it's a modifier on `.s-popover`. I've opted to use styling classes as the source of truth. IMO, the controller kinda reflects that too considering it extends `BasePopoverController`.

I don't really feel like tooltip is its own component as much as it's a modified version of popover. I'm open to ways we can make this less confusing, but I also want to make sure that the file structure reflects components that share a majority of their code (like with `input_textarea` or `checkbox_radio` for example).

Reply:
Makes sense.
I saw you agreed to change the naming convention to the one I suggested so I think the import will become a bit clearer already (`import { PopoverController } from "./popover"`)